recurly/__init__.py

```diff
@@ -813,26 +814,40 @@ def pdf(cls, uuid):
         pdf_response = cls.http_request(url, headers={'Accept': 'application/pdf'})
         return pdf_response.read()
 
-    def refund_amount(self, amount_in_cents, refund_method = 'credit_first'):
-        amount_element = self.refund_open_amount_xml(amount_in_cents,
-                                                     refund_method)
+    def refund_amount(self, amount_in_cents, refund_options = {}):
+        # For backwards compatibility
+        if (isinstance(refund_options, six.string_types)):
+            refund_options = { 'refund_method': refund_options }
+        else:
+            if 'refund_method' not in refund_options:
+                refund_options = { 'refund_method': 'credit_first' }
+
+        amount_element = self._refund_open_amount_xml(amount_in_cents,
+                                                     refund_options)
         return self._create_refund_invoice(amount_element)
 
-    def refund(self, adjustments, refund_method = 'credit_first'):
-        adjustments_element = self.refund_line_items_xml(adjustments,
-                                                         refund_method)
+    def refund(self, adjustments, refund_options = {}):
+        # For backwards compatibility
+        if (isinstance(refund_options, six.string_types)):
+            refund_options = { 'refund_method': refund_options }
+        else:
+            if 'refund_method' not in refund_options:
+                refund_options = { 'refund_method': 'credit_first' }
```

Comment:
I'm not sure. I kind of like it here because it's directly showing how the argument `refund_options` may actually be two different types. I feel like hiding that logic might cause more problems than would help. I think maybe we should have a comment on these that say that the check can be removed one day and then have a note to check the other method.

Reply:
Yeah, a comment would solve the problem I proposed if you think that it would hurt readability to make it a function